### PR TITLE
(Typo): Update README.md

### PR DESCRIPTION
**Pull request by kaleaditya779**
# Description
Corrected the spelling mistakes and increased the grammatical continuity.

**Type of change**
These changes improve the quality of the documentations.  They do not affect the system in any way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +77,11 @@ We are an open-source community-driven project and we love to hear from you. Her
...
-- [distilabeled Intel Orca DPO dataset](https://huggingface.co/datasets/argilla/distilabel-intel-orca-dpo-pairs) used to fine-tune the [improved OpenHermes model](https://huggingface.co/argilla/distilabeled-OpenHermes-2.5-Mistral-7B). This dataset was built by combining human curation in Argilla with AI feedback from distilabel, leading to an improved version of the Intel Orca dataset and outperforming models fine-tuned on the original dataset.
+- [distilled Intel Orca DPO dataset](https://huggingface.co/datasets/argilla/distilabel-intel-orca-dpo-pairs) used to fine-tune the [improved OpenHermes model](https://huggingface.co/argilla/distilabeled-OpenHermes-2.5-Mistral-7B). This dataset was built by combining human curation in Argilla with AI feedback from distilabel, leading to an improved version of the Intel Orca dataset and outperforming models fine-tuned on the original dataset.
```
Comment (davidberenstein1957) [inline README.md]:
```suggestion
- [distilabel Intel Orca DPO dataset](https://huggingface.co/datasets/argilla/distilabel-intel-orca-dpo-pairs) used to fine-tune the [improved OpenHermes model](https://huggingface.co/argilla/distilabeled-OpenHermes-2.5-Mistral-7B). This dataset was built by combining human curation in Argilla with AI feedback from distilabel, leading to an improved version of the Intel Orca dataset and outperforming models fine-tuned on the original dataset.
```

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Argilla can be used for collecting human feedback for a wide variety of AI proje
 
 ### Improve your AI output quality through data quality
 
-Compute is expensive and output quality is important. We help you focus on data, which tackles the root cause of both of these problems at once. Argilla helps you to **achieve and keep high-quality standards** for your data. This means you can improve the quality of your AI output.
+Computational resources are expensive, and output quality is important. We help you focus on data, which tackles the root cause of both of these problems at once. Argilla helps you to **achieve and keep high-quality standards** for your data. This means you can improve the quality of your AI output.
```
Comment (davidberenstein1957) [inline README.md]:
```suggestion
Compute is expensive and output quality is important. We help you focus on data, which tackles the root cause of both of these problems at once. Argilla helps you to **achieve and keep high-quality standards** for your data. This means you can improve the quality of your AI output.
```

**Review by davidberenstein1957 — APPROVED** [review]

**Comment by davidberenstein1957** [comment]
Hi @kaleaditya779 , thanks for the help. It is appreciated 👍

**Comment by frascuchon** [comment]
Thanks for your contribution, @kaleaditya779!

**Comment by kaleaditya779** [comment]
> Thanks for your contribution, @kaleaditya779!

Welcome! It always feels nice when prople like you appreciate such a small efforts from my side.

It is just the start of my open source contributions. 
Hopefully i will soon  contribute to the codebase as well...

**Pull request merged**